src/models/converters/transactions/summary.rs

```diff
@@ -30,7 +30,7 @@ impl Transaction {
             Transaction::Module(transaction) => {
                 Ok(transaction.to_transaction_summary(info_provider))
             }
-            Transaction::Unknown => Err(Error::msg("Unknown transaction type from backend")),
+            Transaction::Unknown => Err(api_error!("Unknown transaction type from backend")),
```

Comment:
In this case we could just as well use the new`bail!`, right?

Reply:
not sure ... just adjusted what was there :P